README.md

```diff
@@ -84,10 +87,27 @@ steps:
       github_token: ${{ github.token }}
 ```
 
-### Buf token
+#### Buf username and Buf API token
+
+Optionally, you can supply both `buf_user` and `buf_api_token` input so that it will perform 
+[Buf Schema Registry][bsr] (BSR) authentication at the same time. This will allow you to access 
+the private remote packages in BSR. This only works when both `buf_user` and `buf_api_token` are
+supplied at the same time:
+
+```yaml
+steps:
+  - uses: bufbuild/buf-setup-action@v1.11.0
+    with:
+      buf_user: ${{ secrets.buf_user }}
+      buf_api_token: ${{ secrets.buf_api_token }}
+```
```

Comment:
```suggestion
#### Buf username and Buf API token

If you are using Private [Remote Packages](https://docs.buf.build/bsr/remote-packages/overview) you may need 
to authenticate the entire system to successfully communicate with the [Buf Schema Registry][bsr]. To achieve this, supply both `buf_user` and `buf_api_token`. This will add your auth credentials to the `.netrc` and allow you to access the BSR from anything in your `PATH`. 

```yaml
steps:
  - uses: bufbuild/buf-setup-action@v1.11.0
    with:
      buf_user: ${{ secrets.buf_user }}
      buf_api_token: ${{ secrets.buf_api_token }}
```
```

Reply:
so the two methods to auth with buf are separated?

Reply:
**buf_token** is a env var instead of an input to buf-setup-action. That's why I want to separate them by "Input" and "Other Configuration". I just changed the section name from "parameter" to "input".